cmd/livepeer_cli/wizard_rounds.go

```diff
@@ -1,11 +1,31 @@
 package main
 
 import (
+	"errors"
 	"fmt"
+	"io/ioutil"
+	"math/big"
+	"net/http"
 )
 
-func (w *wizard) currentRound() string {
-	return httpGet(fmt.Sprintf("http://%v:%v/currentRound", w.host, w.httpPort))
+func (w *wizard) currentRound() (*big.Int, error) {
+	resp, err := http.Get(fmt.Sprintf("http://%v:%v/currentRound", w.host, w.httpPort))
```

Comment:
Eventually it will be nice to update the `httpGet()` helper to include all of this boilerplate and then it can return `([]byte, error)` instead of `string` leaving the conversion of the `[]byte` result up to the caller. Just noting this for the future, but no need to do this now.